### ISSUE TITLE
Disable EBIvariation/vcf-validator

### ISSUE THREAD

**Issue by vladsaveliev** [sigven/pcgr#15]
Hi Sigve,

I'm wondering if there is a way to disable the VCF validation step? I'm struggling picking all the issues with VCFs based on uninformative error messages like `ERROR: Line 218: Info key is not a sequence of alphanumeric and/or punctuation characters` (or even empty messages sometimes!), and validation doesn't really seem to add any value. Would be happy if there were a flag or TOML option that disables the validation for a run, it would make usage of the fantastic tool much more convenient for me.

Thanks
Vlad

**Comment by sigven** [comment]
Hi Vlad,

You are certainly right, this is a good suggestion. Having this as an option in the configuration file is also a good idea. There are in fact multiple users who have struggled with the same issue, and apparently, the VCF format have a number of intricate requirements that most developers are not aware of (and I suspect that a number of tools that accept the VCF file as input are less stringent in their format checking routine). When I noticed that VCF's generated by the ICGC did not validate by vcf-validator, I got somewhat surprised..  On top of that, as you mention, the error messages can sometimes be really hard to comprehend. I will add this functionality in the next release, in which I hope I can also provide support for grch38. Thanks again for the comment!

regards,
Sigve

**Comment by ohofmann** [comment]
Thanks so much, Sigve. If you do end up supporting hg38 dinner and drinks at whatever next conference we run into each other are on me ;-)

**Comment by sigven** [comment]
Deal! Looking forward:-)

**Comment by vladsaveliev** [comment]
Thanks Sigve, looking forward for the new release. Meanwhile I just hotfixed my installation to skip the validation. I figured that the EBI validator even complains about `INFO` flags (e.g. like `cse-hiseq` in `hapmap1=9.0472;cse-hiseq;hapmap2=111.756`), headers like `##SAMPLE=<ID=GBB10_B,Genomes=Germline>` (present in all bcbio-nextgen VCFs), and dots in Integer INFO fields. All those situations are very common. Actually I don't know if there even exist a valid VCF on Earth :-) So easier to turn off EBI validation, and only keep the verification of the fields for potential overlap with the fields added by PCGR.

hg38 support sounds like a dream!

Vlad
